docs/lint.markdown

```diff
@@ -27,6 +27,8 @@ automatically fix many of them.
 
 **The `--fix/-f` option is not supported when passing YAML schemas.**
 
+**The `--json/-j` option prints results as a JSON object rather than human‐readable text.**
```

Comment:
I would remove this line. We don't have it for `inspect` (which also supports the same flag) and I guess it is pretty self explanatory